### ISSUE TITLE
Get the "Fulfillment — basic fee" fee

### ISSUE THREAD

**Issue by yao657355700** [allegro/allegro-api#9537]
Hello
Is there an API interface to obtain this fee "Fulfillment — basic fee"?
﻿I need to get this type of fee for each order through the API interface.
As shown in the picture
![555](https://github.com/user-attachments/assets/01ab2704-f7bf-4849-8b28-f069c61f8de0)

**Comment by PrzemyslawLukanowski** [comment]
Yes, this fee will be returned in the [GET /billing/billing-entries](https://developer.allegro.pl/documentation#tag/Billing) endpoint.

**Comment by yao657355700** [comment]
What is returned here is the total amount per day.
﻿
Is there a way to directly return this cost for a single order?

**Comment by PrzemyslawLukanowski** [comment]
Unfortunately it is not possible.
